libs/gi/page-team/src/TeamSetting/TeamExportModal.tsx

```diff
@@ -0,0 +1,315 @@
+      loadoutData.map(() => defLoadoutExportSetting())
+    )
+  const onExport = () => {
+    const data = database.teams.export(teamId, loadoutDataExportSetting)
+    const dataStr = JSON.stringify(data)
+    navigator.clipboard
+      .writeText(dataStr)
+      .then(() => alert('Copied team data to clipboard.'))
+      .catch(console.error)
+  }
+  const [selEverything, setSelEverything] = useState(true)
+  const unselectEverything = () => {
+    setSelEverything(true)
+    setLoadoutDataExportSetting(
+      loadoutData.map(() => defLoadoutExportSetting())
+    )
+  }
+
+  const selectEverything = () => {
+    setSelEverything(false)
+    setLoadoutDataExportSetting(
+      loadoutData.map((loadoutDatum) => {
+        if (!loadoutDatum) return defLoadoutExportSetting()
+
+        const loadout = database.teamChars.get(loadoutDatum.teamCharId)
+        if (!loadout) return defLoadoutExportSetting()
+        const { buildIds, buildTcIds, customMultiTargets } = loadout
+        return {
+          convertEquipped: true,
+          convertbuilds: buildIds,
+          convertTcBuilds: buildTcIds,
+          exportCustomMultiTarget: customMultiTargets?.map((_, i) => i) ?? [],
+        }
+      })
+    )
+  }
+
+  return (
+    <ModalWrapper open={show} onClose={onHide}>
+      <CardThemed>
+        <CardHeader
+          title="Team Export"
+          action={
+            <IconButton onClick={onHide}>
+              <CloseIcon />
+            </IconButton>
+          }
+        />
+        <Divider />
+        <CardContent sx={{ display: 'flex', flexDirection: 'column', gap: 1 }}>
+          <Alert severity="info">
+            Export the team data to be imported by another user. All the team
+            and loadout data (bonus stats, enemy config, optimize config) are
+            exported. All exported non-TC builds are converted to TC builds.
+          </Alert>
+          <Grid container columns={{ xs: 2, md: 4 }} spacing={1}>
+            {loadoutData.map(
+              (loadout, i) =>
+                !!loadout && (
+                  <Grid item xs={1} key={i}>
+                    <LoadoutSetting
+                      loadout={loadout}
+                      setting={loadoutDataExportSetting[i]}
+                      setSetting={(loadoutExportSetting) =>
+                        setLoadoutDataExportSetting((settings) => {
+                          const data = structuredClone(settings)
+                          data[i] = {
+                            ...data[i],
+                            ...loadoutExportSetting,
+                          }
+                          return data
+                        })
+                      }
+                    />
+                  </Grid>
+                )
+            )}
+          </Grid>
+        </CardContent>
+        <Divider />
+        <CardContent
+          sx={{ display: 'flex', gap: 2, justifyContent: 'flex-end' }}
+        >
+          <Button
```

Comment:
I tried with the check mark, but the check mark/status doesnt actually update as the data becomes partially selected(and detecting it would be more complicated than i care to make it). I can try moving the button up.